packages/terra-menu/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Added inert/aria-hidden to prevent screen reader from moving focus out of popup.
```

Comment:
I would imagine that the change could be moved to terra-popup for reuse within terra-menu and terra-date-picker, but I'm unsure about replacing the functionality with terra-overlay.

Unfortunately, while I don't think that it would hurt to use this functionality in terra-popup for a basic use-case, it isn't going to solve the problem when it's presented from a modal as this implementation only affects #root.

@bjankord How do we want to address layered portal'd elements? I imagine the most common use-case for this is going to be terra-collapsible-menu-view within headers of modals. The other likely layer scenario would be the terra-notification-dialog paired with modals and dirty data.